samcli/commands/local/lib/local_api_service.py

```diff
@@ -63,6 +63,7 @@ def start(self):
             port=self.port,
             host=self.host,
             stderr=self.stderr_stream,
+            disable_authorizer=self.disable_authorizer
```

Comment:
Is `self.disable_authorizer` defined? It looks like it was passed into the constructor, but wasn't saved anywhere.